### PR TITLE
feat: omit prefix symbol automatically when copying shell code

### PR DESCRIPTION
**Pull request by codersjj**
### 🤔 这个变动的性质是？/ What is the nature of this change?

- [x] 新特性提交 / New feature
- [ ] bug 修复 / Fix bug
- [ ] 样式优化 / Style optimization
- [ ] 代码风格优化 / Code style optimization
- [ ] 性能优化 / Performance optimization
- [ ] 构建优化 / Build optimization
- [ ] 网站、文档、Demo 改进 / Website, documentation, demo improvements
- [ ] 重构代码或样式 / Refactor code or style
- [ ] 测试相关 / Test related
- [ ] 其他 / Other

### 💡 需求背景和解决方案 / Background or solution

Taking [Ant Design's docs](https://ant.design/docs/react/use-with-create-react-app#import-antd) as an example:

```bash
$ npm install antd --save
```

When you click the "Copy" button, the result will be:

`$ npm install antd --save`

After applying the code from this commit, the result will be:

`npm install antd --save`

This is more in line with expectations.

### 📝 更新日志 / Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | remove leading spaces and specific characters when copying shell code in `SourceCode` |
| 🇨🇳 Chinese | 在 `SourceCode` 中复制 shell 代码时，去除行首的空格和特定字符 |

### CODE REVIEW

File: src/client/theme-default/builtins/SourceCode/index.tsx [hunk 1]
```diff
@@ -28,10 +28,16 @@ const SourceCode: FC<SourceCodeProps> = (props) => {
   const [isCopied, setIsCopied] = useState(false);
   const { themeConfig } = useSiteData();
 
+  let text = '';
+  const isShell = /shellscript|shell|bash|sh|zsh/.test(lang);
```
Comment (PeachScript) [inline src/client/theme-default/builtins/SourceCode/index.tsx:32]: 套个 `useEffect`？

Comment (codersjj) [reply src/client/theme-default/builtins/SourceCode/index.tsx]: Wrapped

File: src/client/theme-default/builtins/SourceCode/index.tsx [hunk 2]
```diff
@@ -28,10 +28,18 @@ const SourceCode: FC<SourceCodeProps> = (props) => {
   const [isCopied, setIsCopied] = useState(false);
   const { themeConfig } = useSiteData();
 
+  let text = '';
```
Comment (PeachScript) [inline src/client/theme-default/builtins/SourceCode/index.tsx:31]: `text` 需用 state 存储，`children` 需加到 useEffect 的依赖里？

Comment (codersjj) [reply src/client/theme-default/builtins/SourceCode/index.tsx]: Sorry, I made some mistakes here, have committed a new fix.

**Review by PeachScript — APPROVED** [review]
+1

**Pull request merged**